@ecomplus/storefront-template/template/scss/themes/beauty/_variables.scss

```
@@ -0,0 +1,78 @@
+$icons-font: "line-awesome" !default;
+
+// Beauty E-Com
+
+//
+// Color system
+//
+
+$white:    #fff !default;
+$gray-100: #f8f9fa !default;
+$gray-200: #e9ecef !default;
+$gray-300: #d9d9d9 !default;
+$gray-400: #ced4da !default;
+$gray-500: #adb5bd !default;
+$gray-600: #9A9A9A !default;
+$gray-700: #495057 !default;
+$gray-800: #343a40 !default;
+$gray-900: #212529 !default;
+$black:    #000 !default;
+
+$blue:           #033c73 !default;
+$indigo:         #6610f2 !default;
+$purple:         #6f42c1 !default;
+$purple-dark:    #37003c !default;
+$grayish-purple: #5F506B !default;
+$purple-light:   #C0B9DD !default;
+$pink-baby:      #E3D0D8 !default;
+$salmon-pink:    #D88C88 !default;
+$pink:           #e83e8d !default;
```

```suggestion
$pink:           #E83E8C !default;
```

É esse o padrão do tema, certo?